paper/paper.md

```diff
@@ -31,21 +31,21 @@ StateMint includes a Mathematica package, a Python package, and a web interface
 
 # Introduction
 
-When deriving a system's state-space model&mdash;that is, the vector state (differential) equation and the vector output (algebraic) equation&mdash;one begins by forming scalar equations for each element describing its dynamics.
+When deriving a system's state-space model&mdash;that is, the vector state (differential) equation and the vector output (algebraic) equation&mdash;one begins by forming scalar equations for each element **REVIEW: Not all systems are "element"-based. The reader may need more background here.** describing its dynamics.
 The next step is to form a set of $N$ constraint equations that describe the topology of the system defined by the interconnection of the $N$ elements.
-A set of $2N$ differential and algebraic equations and $2N$ unknown variables result.
-If properly constructed (e.g. with the linear graph technique), $N$ of the unknown variables can be immediately eliminated through direct substitution.
+A set of $2N$ differential and algebraic equations and $2N$ unknown variables result **REVIEW: Add a citation for why this is the case?**.
```

Comment:
This is the case because each element has its elemental equation, along with a continuity or compatibility (a constraint) equation. Each element also has a through and across-variable associated with it. I have not seen this explicitly stated anywhere, but it can be derived from the linear graph methods. If you would like I could reference Rowell and Wormley here.